src/screen.rs

```diff
@@ -1078,10 +1078,30 @@ impl Screen {
         self.need_clear_lines = false;
         self.need_clear_screen = false;
 
+        // Compare the number of lines in the (new) left prompt compared to the
```

Comment:
PR description should go into commit message (notably the "Fixes
#ISSUE_NUMBER" would be useful to autoclose the issue if the PR gets
pushed without the github-native merge methods)